_posts/2020-01-13-pairing-metrics.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "Pairing metrics"
+date: 2020-01-13
+permalink: "/pairing-metrics"
+description: >-
+  Don't be misled by your data
+---
+
+What you're measuring doesn't makes sense on its own. For a SaaS product, a high number of registrations doesn't mean success. A high number of registrations, low churn and turning a profit means success.
+
+As Andrew Grove's speaks in High Output Management, it's pivotal to pair metrics to avoid over optimising your business in the wrong ways:
```

Comment:
```suggestion
As Andrew Grove's says in [High Output Management](https://www.amazon.co.uk/High-Output-Management-Andrew-Grove/dp/0679762884/ref=sr_1_1?crid=3FEWRXR5M6SJU&keywords=high+output+management&qid=1573592913&s=books&sprefix=high+%2Cstripbooks%2C139&sr=1-1), it's pivotal to pair metrics to avoid over optimising your business in the wrong ways:
```